models/models.js

```diff
@@ -42,3 +42,38 @@ exports.selectArticleByID = (article_id) => {
     return article;
   });
 };
+
+exports.selectCommentsByArticleID = (article_id) => {
+  let SQL = `
+    SELECT comments.comment_id, comments.votes, comments.created_at, comments.author, comments.body
+    FROM comments
+    JOIN articles
+    ON comments.article_id = articles.article_id
+    WHERE comments.article_id = $1
+    GROUP BY comments.comment_id
+    ORDER BY comments.created_at 
```

Comment:
Im pretty sure the comments have an article id that you can just search for, im not sure what the JOIN is doing.